### PR TITLE
Add returns block to level_populations() doc

### PR DESCRIPTION
**Pull request by dstansby**
It would be nice eventually if this 3D array came with indexes (ie. an xarray? a 3D astropy table?)

### CODE REVIEW

File: fiasco/ion.py [hunk 1]
```diff
@@ -254,6 +254,13 @@ def level_populations(self,
         density : `~astropy.units.Quantity`
         include_protons : `bool`, optional
             If True (default), include proton excitation and de-excitation rates
+
+        Returns
+        -------
+        `~astropy.units.Quantity`
+            A ``(l, m, n)`` shaped quantity, where ``l`` is the number number of
+            temperatures, ``m`` is the number of densities, and ``n``
+            is the energy level.
```
Comment (wtbarnes) [inline fiasco/ion.py]:
```suggestion
            A ``(l, m, n)`` shaped quantity, where ``l`` is the number of
            temperatures, ``m`` is the number of densities, and ``n``
            is the number of energy levels.
```

**Comment by wtbarnes** [comment]
Thanks for this! I fixed a minor typo. I didn't actually mean to commit it (:sweat_smile:), but it was a very small change.

**Comment by dstansby** [comment]
Oh I think I committed it, presumably straight after you left it!

**Comment by wtbarnes** [comment]
> It would be nice eventually if this 3D array came with indexes (ie. an xarray? a 3D astropy table?)

Good point. Returning a 3D array without any context as to what the dimensions correspond to is not very user friendly. An xarray is a good idea. That could be useful for the contribution function, level population, intensity too...

**Pull request merged**